src/main/java/uk/gov/hmcts/darts/arm/rpo/impl/ArmRpoApiImpl.java

```diff
@@ -277,6 +295,13 @@ private ArmRpoException handleFailureAndCreateException(String message,
         return new ArmRpoException(message);
     }
 
+    private static StorageAccountRequest createStorageAccountRequest() {
```

Comment:
Can be non-static for consistency with the other private methods.

Reply:
Good spot, I don't know why when I refactor code it automatically adds static each time